### PR TITLE
026 add branding to the product

### PR DESCRIPTION
**Pull request by omcmenamin**

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -31,5 +30,8 @@ app.listen(3000, ()=> {
 });
 
 app.get('/', (req, res) => {
-  res.send('hello world')
-})
+  res.render('index')
+});
+app.get('/academy', (req, res) => {
+  res.render('academy')
+});
```
Comment (shaunganley) [inline app.ts:37]: this doesn't look necessary

File: views/academy.html
Comment (shaunganley) [inline views/academy.html:1]: is this file needed?

**Review by shaunganley — APPROVED** [review]

File: app.ts (hunk 1 above)
Comment (omcmenamin) [reply app.ts:37]: needed to access front end page as its not GETting anything from API

**Review by SeanMcAvoy29 — APPROVED** [review]
